packages/mask/src/plugins/Wallet/SNSAdaptor/GasSettingDialog/GasSetting1559.tsx

```diff
@@ -199,7 +199,7 @@ export const GasSetting1559: FC<GasSettingProps> = memo(
         const handleConfirm = useCallback(
             (data: zod.infer<typeof schema>) => {
                 onConfirm?.({
-                    gasLimit: data.gasLimit,
+                    gasLimit: data.gasLimit as any,
```

Comment:
I don't know why this isn't a type error before. But data.gasLimit is a string, but gasLimit in onConfirm is number. I won't touch it in this PR. cc @guanbinrui